### ISSUE TITLE
`init` fails with openssl 1.1.0

### ISSUE THREAD

**Issue by krisleech** [shadowhand/git-encrypt#52]
Using the latest version of gitcrypt (legacy branch).
I have a repo which already has gitcrypte'd files which I have cloned to another machine.
When I use `gitcrypt init` I enter the salt and password but choosing the cipher fails.

```
gitcrypt init   
<SNIP>
What encryption cipher do you want to use? [aes-256-ecb] 
Invalid command 'list-cipher-commands'; type "help" for a list.
Error: Cipher 'aes-256-ecb' is not available
```

`Invalid command 'list-cipher-commands'; type "help" for a list.` is the error and this seems to be coming from `openssl` at [this line](https://github.com/shadowhand/git-encrypt/blob/legacy/gitcrypt#L79).

* OpenSSL 1.1.0g  2 Nov 2017
* gitcrypt (legacy branch)
* Ubuntu 18.04.1 LTS

**Comment by krisleech** [comment]
I can hack around this by inserting a `break` above `local exists`. The script exists without errors but the files do not decrypt.

**Comment by krisleech** [comment]
For anyone wanting to recover their files you can manually decrypt as such:
`cat the-file.encrypted | openssl aes-256-ecb -a -salt`

**Comment by shadowhand** [comment]
Please do not use this package for new projects. See [the README](https://github.com/shadowhand/git-encrypt/blob/master/README.md).

This error is occurring because `aes-256-ecb` is not a safe cipher.

**Comment by krisleech** [comment]
It isn't a new project, we are already using git-encrypt. I cloned on to a new machine, thus I needed to setup git-encrypt. I only found out the project was discontinued today. I've found a way decrypt the files without git-encrypt so I can now get the plain text files and move on to another system such as the one suggested in the README. Thanks for the maintaining this project until now :+1: it has been very useful.
